src/components/Authentication/Signup.jsx

```diff
@@ -0,0 +1,145 @@
+                                setUserInput({ ...userInput, lastName: event.target.value })
+                            }
+                        />
+                    </div>
+
+                    <div className="form-input my-4 flex flex-col">
+                        <label for="login-email" className="input-label text-left text-lg">
+                            {' '}
+                            Email:{' '}
+                        </label>
+                        <input
+                            type="text"
+                            required
+                            placeholder="Enter your email address"
+                            name="login-email"
+                            className="login-email input-form border-[1px] border-color-grey rounded p-1"
+                            onChange={(event) =>
+                                setUserInput({ ...userInput, email: event.target.value })
+                            }
+                        />
+                    </div>
+                    
+                    <div className="form-input my-4 flex flex-col">
+                        <label htmlFor="login-username" className="input-label text-left text-lg">
+                            Username:
+                        </label>
+                        <input
+                            type="text"
+                            placeholder="Enter your username"
+                            name="login-username"
+                            required
+                            className="login-username input-form border-[1px] border-color-grey rounded p-1"
+                            onChange={(e) =>
+                                setUserInput({ ...userInput, username: e.target.value })
+                            }
+                        />
+                    </div>
+
+                    <div className="form-input my-4 flex flex-col">
+                        <label for="login-password" className="input-label text-left text-lg">
+                            {' '}
+                            Passsword:{' '}
+                        </label>
+                        <input
+                            type="password"
+                            required
+                            placeholder="Enter your password"
+                            name="login-password"
+                            className="login-password input-form border-[1px] border-color-grey rounded p-1"
+                            onChange={(event) =>
+                                setUserInput({ ...userInput, password: event.target.value })
+                            }
+                        />
+                    </div>
+
+                    <div className="form-input checkbox-reset">
+                        <div>
+                            <input type="checkbox" name="form-checkbox" className="form-checkbox" />
+                            <label for="form-checkbox" className="checkbox-label">
```

Comment:
For label, I think you need to use `htmlFor` attribute, not the `for` attribute

Reply:
Thanks for pointing it out. I'll make the changes.